fearless/ApplicationLayer/Services/WalletBalanceSubscription/WalletBalanceBuilder.swift

```diff
@@ -122,11 +122,7 @@ final class WalletBalanceBuilder: WalletBalanceBuilderProtocol {
         var disabledChainAssets: [ChainAsset] = []
 
         chainAssets.forEach { chainAsset in
-            let accountRequest = chainAsset.chain.accountRequest()
-            guard let accountId = metaAccount.fetch(for: accountRequest)?.accountId else {
-                return
-            }
-            let chainAssetKey = chainAsset.uniqueKey(accountId: accountId)
+            let chainAssetKey = chainAsset.uniqueKey(accountId: metaAccount.substrateAccountId)
```

Comment:
please use correct account id (wallet.fetch...accountId) instead of hardcoded substrate